Sources/MongoSwift/BSON/Document.swift

```diff
@@ -205,7 +205,8 @@ public struct Document: ExpressibleByDictionaryLiteral, ExpressibleByArrayLitera
         }
     }
 
-    /// Sets key to newValue. if checkForKey=false, the key/value pair will be appended without checking for the key's presence first.
+    /// Sets key to newValue. if checkForKey=false, the key/value pair will be appended without checking for the key's
+    // presence first.
```

Comment:
```suggestion
    /// presence first.
```
although this is internal, we try to follow the same docstring policies as public types anyway.

Reply:
Right, I recall this from the `Development.md` but forgot about this. Done.